lib/cli.rb

```diff
@@ -367,7 +360,14 @@ def self.check_port_validity(port)
       name1 = shift_arg(args, "NAME1")
       name2 = shift_arg(args, "NAME2")
 
-      check_port_validity(options[:port]) if options[:html]
+      if options[:html]
+        begin
+          check_port_validity(@config.http_server_port)
+        rescue Machinery::Errors::ServerPortError => e
+          raise Machinery::Errors::InvalidCommandLine.new(e.message +  " The port can be " \
```

Comment:
Unnecessary spacing detected.<br>Operator `+` should be surrounded with a single space.